conf/application.conf

```diff
@@ -145,6 +145,12 @@ contact-frontend {
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:6791/customs-declare-exports/start"
+
+  customsDeclarationsGoodsTakenOutOfEu = "https://www.gov.uk/guidance/customs-declarations-for-goods-taken-out-of-the-eu"
+  commodityCodes = "https://www.gov.uk/guidance/finding-commodity-codes-for-imports-or-exports"
+  relevantLicenses = "https://www.gov.uk/starting-to-export/licences"
+  serviceAvailability = "https://www.gov.uk/guidance/customs-declaration-service-service-availability-and-issues"
+  customsMovementsFrontend = "http://localhost:6796/customs-movements/start"
```

Comment:
This must be overridden at least in `appConfigBase`, please prepare PR

Reply:
PR ready.